src/main/webapp/app/guided-tour/guided-tour-step.model.ts

```diff
@@ -0,0 +1,42 @@
+import { ContentType, LinkType, Orientation, OrientationConfiguration } from 'app/guided-tour/guided-tour.constants';
+
+export interface TourStep {
```

Comment:
note: for now this is fine. In the future I would like to see a taxonomy here where TourStep is an abstract class and includes several subtypes, e.g. TextTourStep, PictureTourStep, TextTourStep, etc.

Reply:
With this we could then avoid most of the `?` in the type definitions